commit 83f7e4556cf3b0f94bda06b22c4e9ef3266a9d15
Author: Jeff Billimek <jeff@billimek.com>
Date:   Fri Jul 26 23:55:53 2019 -0400

    Frigate (#51)
    
    * frigate helm chart
    
    Signed-off-by: Jeff Billimek <jeff@billimek.com>
    
    * remove trailing spaces
    
    * bump chart version
    
    * refactoring frigate ingress
    
    * refactoring configmap
    
    * refactoring configmap

diff --git a/frigate/Chart.yaml b/frigate/Chart.yaml
index 6df1973e..2d7200cf 100644
--- a/frigate/Chart.yaml
+++ b/frigate/Chart.yaml
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.2.0-beta"
 description: Realtime object detection on RTSP cameras with the Google Coral
 name: frigate
-version: 0.1.2
+version: 0.1.3
 keywords:
   - tensorflow
   - coral
diff --git a/frigate/templates/NOTES.txt b/frigate/templates/NOTES.txt
index edbc9f35..17e6e6ed 100644
--- a/frigate/templates/NOTES.txt
+++ b/frigate/templates/NOTES.txt
@@ -1,9 +1,7 @@
 1. Get the application URL by running these commands:
 {{- if .Values.ingress.enabled }}
-{{- range $host := .Values.ingress.hosts }}
-  {{- range .paths }}
-  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ $host.host }}{{ . }}
-  {{- end }}
+{{- range .Values.ingress.hosts }}
+  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ . }}{{ $.Values.ingress.path }}
 {{- end }}
 {{- else if contains "NodePort" .Values.service.type }}
   export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "frigate.fullname" . }})
@@ -11,11 +9,11 @@
   echo http://$NODE_IP:$NODE_PORT
 {{- else if contains "LoadBalancer" .Values.service.type }}
      NOTE: It may take a few minutes for the LoadBalancer IP to be available.
-           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "frigate.fullname" . }}'
+           You can watch the status of by running 'kubectl get svc -w {{ include "frigate.fullname" . }}'
   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "frigate.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
   echo http://$SERVICE_IP:{{ .Values.service.port }}
 {{- else if contains "ClusterIP" .Values.service.type }}
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "frigate.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
-  echo "Visit http://127.0.0.1:8080 to use your application"
-  kubectl port-forward $POD_NAME 8080:80
+  echo "Visit http://127.0.0.1:5000 to use your application"
+  kubectl port-forward $POD_NAME 5000:5000
 {{- end }}
diff --git a/frigate/templates/configmap.yaml b/frigate/templates/configmap.yaml
index 6733fb85..d58f50a1 100644
--- a/frigate/templates/configmap.yaml
+++ b/frigate/templates/configmap.yaml
@@ -10,4 +10,4 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
   config.yml: |
-{{ toYaml .Values.config | default "{}" | indent 4 }}
+{{ .Values.config | indent 4 }}
diff --git a/frigate/values.yaml b/frigate/values.yaml
index a40ee6db..517abeeb 100644
--- a/frigate/values.yaml
+++ b/frigate/values.yaml
@@ -19,7 +19,7 @@ nameOverride: ""
 fullnameOverride: ""
 
 # frigate configuration - see https://github.com/blakeblackshear/frigate/blob/master/config/config.yml for example
-config:
+config: |
   mqtt:
     host: mqtt.server.com
     topic_prefix: frigate
